src/utils/test/validation.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { validateComponentTag } from '../validation';
+
+describe('validation', () => {
+  it.each([' my-tag', 'my-tag ', ' my-tag '])('should error on whitespace', (tagName) => {
```

Comment:
Minor, can we add an additional suite/`describe` block here? I'd like to get the code to start following the general pattern of
```js
describe('filename-under-test', () => {
  describe('function-or-class-under-test', () => {
    // any other necessary suites to split up the code
      // tests
  });
});
```
(which we don't have uniformly across the codebase today).

The reason for which is that as these files grow in size/scope, it makes adding additional suites easier (in that there's less churn in adding whitespace when we need to add a new `describe` block for a new function).

```suggestion
  describe('validateComponentTag', () => {
    ... existing tests, indented by 2 spaces
```